### PR TITLE
ECS migration changes to Buyer UI

### PR DESCRIPTION
**Pull request by mcbhenwood**
Changes to facilitate provision of Buyer UI in ECS Fargate. Please review by commit for clarity.

### CODE REVIEW

File: src/main/setup/routes/setupHomeRoutes.ts
Comment (mcbhenwood) [inline src/main/setup/routes/setupHomeRoutes.ts]: `setupHomeRoutes.ts` isn't a natural home for a healthcheck route, however I noted that the existing `/health` route is mounted here and so I've followed in the footsteps of that precedent.

File: src/main/routes/isAlive.ts
Comment (mcbhenwood) [inline src/main/routes/isAlive.ts]: A deliberately simple, instant, "is there a server process alive?" healthcheck - The existing one is too weighty and contains too many dependencies which could lead to churn in the ECS tasks

File: src/main/setup/redis/client.ts
Comment (mcbhenwood) [inline src/main/setup/redis/client.ts:43]: Inside our VPC we do not need TLS for Redis.

**Comment by NToppingCCS** [comment]
No problems with what I see here, though it's worth noting the linting appears to have a couple of issues with it which will need resolving.

I will hold off on approving until both that is resolved, and we're in a place where we're able to merge this.

**Review by NToppingCCS — CHANGES_REQUESTED** [review]
@mcbhenwood looks like the build is failing on this one.  Could you bring the branch up to date and resolve that?  Once that's done, happy to approve.

**Comment by mcbhenwood** [comment]
> @mcbhenwood looks like the build is failing on this one. Could you bring the branch up to date and resolve that? Once that's done, happy to approve.

Apols, I missed the failure somehow - linted and green @NToppingCCS

**Review by tim-s-ccs — APPROVED** [review]

**Review by NToppingCCS — APPROVED** [review]
Good to go now :)

**Pull request merged**
